pages/post.tsx

```diff
@@ -56,25 +39,43 @@ const post = () => {
     setActiveStep((prevActiveStep) => prevActiveStep - 1)
   }
 
-  //   const handleSkip = () => {
-  //     if (!isStepOptional(activeStep)) {
-  //       // You probably want to guard against something like this,
-  //       // it should never occur unless someone's actively trying to break something.
-  //       throw new Error("You can't skip a step that isn't optional.")
-  //     }
+  //step1 EventTitle 상태 로직
+  const [eventTitle, setEventTitle] = useState('')
+  const [participant, setParticipant] = useState<number>()
+  const [inputState, setInputState] = useState<string>('')
 
-  //     setActiveStep((prevActiveStep) => prevActiveStep + 1)
-  //     setSkipped((prevSkipped) => {
-  //       const newSkipped = new Set(prevSkipped.values())
-  //       newSkipped.add(activeStep)
+  const handleInput = (e: React.ChangeEvent<HTMLInputElement>) => {
+    e.target.name === 'eventTitle'
+      ? setEventTitle(() => e.target.value)
+      : setParticipant(() => Number(e.target.value))
+  }
 
-  //       return newSkipped
-  //     })
-  //   }
+  const handleRaido = (e: React.ChangeEvent<HTMLInputElement>) => {
+    setInputState(() => e.target.value)
+  }
 
-  //   const handleReset = () => {
-  //     setActiveStep(0)
-  //   }
+  // step별 컴포넌트 로직
+  const getStepContent = (stepNumber: number) => {
+    switch (stepNumber) {
+      case 0:
+        return (
+          <EventTitle
+            eventTitle={eventTitle}
+            participant={participant}
+            inputState={inputState}
+            handleInput={handleInput}
+            handleRaido={handleRaido}></EventTitle>
```

Comment:
```suggestion
            handleRaido={handleRaido} />
```
닫는 태그로 추후 작업하시면서 다듬어주시면 더 좋을 것 같습니다!